etl_manager/etl.py

```diff
@@ -385,12 +404,54 @@ def job_run_state(self) :
 
         return status['JobRun']['JobRunState'].lower()
 
+    def wait_completion(self):
```

Comment:
would the name `wait_for_completion` be better?

Reply:
🤷‍♂️ - I had the same thought and then I just ended up calling in a similar way to what they called it in the Glue operator ([`job_completion()`](https://github.com/apache/incubator-airflow/pull/3504/files#diff-31562106f0b6b3c8d91f1fb7b2f6bcb2R135), which BTW I think has an infinite loop 🤣 )

Happy to change if you think it's better.

Reply:
I think `wait_for_completion` is clearer/easier to understand and worth making a change

Reply:
I'll change then, no problem 👍